src/Elcodi/Component/User/Entity/Abstracts/AbstractUser.php

```diff
@@ -74,11 +89,11 @@
     protected $lastname;
 
     /**
-     * @var int
+     * User gender, by default undefined.
      *
-     * gender
+     * @var int
      */
-    protected $gender;
+    protected $gender = self::GENDER_UNDEFINED;
```

Comment:
the constant is used here to keep the entity consistent, so there's no possibility to have a `null` or a `GENDER_UNDEFINED` value, at least there's the `GENDER_UNDEFINED` value.